entries/global-alert.ts

```diff
@@ -0,0 +1 @@
+export * from "components/globalAlert/GlobalAlert";
```

Comment:
Can we actually move this to the entries/notifications? Eventually, I'll have to move the other "Alert" component to that module as well.